uber/decorators.py

```diff
@@ -458,11 +458,11 @@ def with_rendering(*args, **kwargs):
             result = func(*args, **kwargs)
         except CSRFException as e:
             message = "Your CSRF token is invalid. Please go back and try again."
-            uber.server.log_exception_with_verbose_context(str(e))
+            uber.server.log_exception_with_verbose_context(msg=str(e))
```

Comment:
This is just personal preference, but I feel like `msg` should be the first parameter in the function, so we don't have to specify the `msg` keyword. That's pretty much a de facto standard for printing/logging functions. In fact, if you'll notice, that exactly how we declare `log_with_verbose_context()`:

```python
def log_with_verbose_context(msg, exc_info=False):
    full_msg = '\n'.join([msg, get_verbose_request_context()])
    log.error(full_msg, exc_info=exc_info)


def log_exception_with_verbose_context(debug=False, msg=''):
    """
    Write the request headers, session params, page location, and the last error's traceback to the cherrypy error log.
    Do this all one line so all the information can be collected by external log collectors and easily displayed.

    Debug param is there to play nice with the cherrypy logger
    """
    log_with_verbose_context('\n'.join([msg, 'Exception encountered']), exc_info=True)
```

I know you left a comment indicating the `debug` parameter is there for the cherrypy logger, but can you elaborate on that? Does it need to be the first parameter, or would it work as well if it's the second parameter?